app/mainwindow.cpp

```diff
@@ -313,10 +313,12 @@ void MainWindow::fileExportToHtml()
 
 void MainWindow::fileExportToPdf()
 {
-    ExportPdfDialog dialog(fileName);
-    if (dialog.exec() == QDialog::Accepted) {
-        ui->webView->print(dialog.printer());
-    }
+	ExportPdfDialog dialog(fileName);
+	if (dialog.exec() == QDialog::Accepted) {
+		 QTextDocument doc;
```

Comment:
Interesting that this fixes the problem. Did you just tried this out or do you have any relevant links for this like e.g. Qt bugreport or a stackoverflow page?

Do we lose anything with this fix because of the [HTML subset](http://doc.qt.io/qt-5/richtext-html-subset.html) that Qt uses? Or is this subset only relevant for the widgets that display the QTextDocument?